llvm/test/CodeGen/RISCV/GlobalISel/instruction-select/shift-rv64.mir

```diff
@@ -241,3 +241,89 @@ body:             |
     $x10 = COPY %6(s64)
     PseudoRET implicit $x10
 ...
+
+---
+name:            srl_and_needed
+alignment:       2
+legalized:       true
+regBankSelected: true
+tracksRegLiveness: true
+registers:
```

Comment:
I think you can drop this `registers` section since all registers have their register class specified below (i.e. `%0:gprb(s64)`)

Reply:
Good suggestion, thanks